web/tests/integration/user.test.ts

```diff
@@ -253,10 +253,10 @@ describe("user role", () => {
 
     const response = await client.query({
       query,
+
       variables: {
         id: userFromTeam1?.user_id,
       },
-      context: { headers: { team_id: teams[1].id } }, // Pass a malicious header
```

Comment:
this test will pass now because we are not spoofing anything. This test doesn't cover anything anymore

Reply:
Removed this one